src/Uno.UI/UI/Xaml/Controls/ComboBox/ComboBox.cs

```diff
@@ -382,11 +382,6 @@ private void UpdateContentPresenter()
 					_contentPresenter.Content = b.Value;
 				}
 
-				if (itemView != null && itemView.GetVisualTreeParent() != _contentPresenter)
-				{
-					// Item may have been put back in list, reattach it to _contentPresenter
-					_contentPresenter.AddChild(itemView);
-				}
```

Comment:
I'm not sure why this was present in the first place. I didn't find a matching runtime test or something, so I'm just waiting to see what might break in CI. When these lines are present, the _contentPresenter actually has 2 children (and WinUI doesn't do that).